reward/reward_distributor.go

```diff
@@ -53,8 +53,16 @@ func NewRewardDistributor(gh governanceHelper) *RewardDistributor {
 // getTotalTxFee returns the total transaction gas fee of the block.
 func (rd *RewardDistributor) getTotalTxFee(header *types.Header, rewardConfig *rewardConfig) *big.Int {
 	totalGasUsed := big.NewInt(0).SetUint64(header.GasUsed)
-	totalTxFee := totalGasUsed.Mul(totalGasUsed, rewardConfig.unitPrice)
-	return totalTxFee
+	if header.BaseFee != nil {
+		// kip71 hardfork
+		return totalGasUsed.Mul(totalGasUsed, header.BaseFee)
+	} else {
+		return totalGasUsed.Mul(totalGasUsed, rewardConfig.unitPrice)
+	}
+}
+
+func (rd *RewardDistributor) txFeeBurning(txFee *big.Int) *big.Int {
+	return txFee.Div(txFee, big.NewInt(2))
```

Comment:
Do you plan to not manage the burning rate(which is 50% now) as a separate governance variable?

Reply:
No It doesn't. It is determined by @lwj010

Reply:
@jimni1222 Yes, it's intentionally omitted for governance variable - since it's always good for validators to choose not to burn at all.

Reply:
Oh okay. i understand. Thank you for your explaination.